### ISSUE TITLE
The predict result problem

### ISSUE THREAD

**Issue by er778899789** [Tianxiaomo/pytorch-YOLOv4#396]
Whwn I finish training the model with yolov4,and the result is look pretty good.
I use it to test my test datasets, the accuracy is nice but sometimes will happend double recognize.
What's the reason to cause the problem?

![擷取](https://user-images.githubusercontent.com/32409355/109577284-533bb280-7b30-11eb-9e45-f6e5e6c30419.PNG)

**Comment by sharoseali** [comment]
@er778899789  can you please tell for how much epochs you train on pytorch yolo v4 to get  enough accuracy and how you measure the it on test / val set

**Comment by batuhanbeytekin** [comment]
Hİ! I want to predict my training result but when I visualized my test set, bboxes are not visible. How did you preaper your train.txt?

**Comment by er778899789** [comment]
@sharoseali I solve the problem and forgot to close. THX

**Comment by er778899789** [comment]
@batuhanbeytekin Hi! train.txt just label the images with yolo format.
